src/solvers/thermal_operators.hpp

```diff
@@ -15,119 +21,135 @@
 namespace serac {
 
 /**
- * The time dependent operator for advancing the discretized conduction ODE
+ * @brief The time dependent operator for advancing the discretized conduction ODE
  */
 class DynamicConductionOperator : public mfem::TimeDependentOperator {
- protected:
+protected:
   /**
-   * Finite Element space
+   * @brief Finite Element space
    */
   std::shared_ptr<mfem::ParFiniteElementSpace> fespace_;
 
   /**
-   * Grid function for boundary condition projection
+   * @brief Grid function for boundary condition projection
    */
   std::shared_ptr<mfem::ParGridFunction> state_gf_;
 
   /**
-   * Solver for the mass matrix
+   * @brief Solver for the mass matrix
    */
   std::unique_ptr<mfem::CGSolver> M_solver_;
 
   /**
-   * Solver for the T matrix
+   * @brief Solver for the T matrix
    */
   std::unique_ptr<mfem::CGSolver> T_solver_;
 
   /**
-   * Preconditioner for the M matrix
+   * @brief Preconditioner for the M matrix
    */
   std::unique_ptr<mfem::HypreSmoother> M_prec_;
 
   /**
-   * Preconditioner for the T matrix
+   * @brief Preconditioner for the T matrix
    */
   std::unique_ptr<mfem::HypreSmoother> T_prec_;
 
   /**
-   * Pointer to the assembled M matrix
+   * @brief Pointer to the assembled M matrix
    */
   std::shared_ptr<mfem::HypreParMatrix> M_mat_;
 
   /**
-   * Pointer to the assembled K matrix
+   * @brief Pointer to the assembled K matrix
    */
   std::shared_ptr<mfem::HypreParMatrix> K_mat_;
 
   /**
-   * Pointer to the assembled T ( = M + dt K) matrix
+   * @brief Pointer to the assembled T ( = M + dt K) matrix
    */
   std::unique_ptr<mfem::HypreParMatrix> T_mat_;
 
   /**
-   * Pointer to the eliminated T matrix
+   * @brief Pointer to the eliminated T matrix
    */
   std::unique_ptr<mfem::HypreParMatrix> T_e_mat_;
 
   /**
-   * Assembled RHS vector
+   * @brief Assembled RHS vector
    */
   std::shared_ptr<mfem::Vector> rhs_;
 
   /**
-   * RHS vector including essential boundary elimination
+   * @brief RHS vector including essential boundary elimination
    */
   std::shared_ptr<mfem::Vector> bc_rhs_;
 
   /**
-   * Temperature essential boundary coefficient
+   * @brief Temperature essential boundary coefficient
    */
   std::vector<serac::BoundaryCondition>& ess_bdr_;
 
   /**
-   * Auxillary working vectors
+   * @brief Auxillary working vectors
    */
-  mutable mfem::Vector z_;
-  mutable mfem::Vector y_;
-  mutable mfem::Vector x_;
+  mutable mfem::Vector z_, y_, x_;
```

Comment:
ISOCPP [generally advises](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-name-one) to keep one declaration per line, though I'm not especially bothered by the use of the comma operator here.

Reply:
I'm OK either way. It's probably best to keep them on separate lines.